src/components/ListReference/index.tsx

```diff
@@ -1,11 +1,134 @@
+const pvpCombatPath = "https://drive.google.com/uc?export=view&id=1SH0lLwb11g3JQ6OCRmB5MvKvRl1EEodo";
+const guideToFirstGamePath = "https://drive.google.com/uc?export=view&id=1lIL4cNGS7yJCSXGIl6e22UnSxchw8_mc";
+
+type Contents =
+  | "list"
+  | "referenceSheet"
+  | "monsterReference"
+  | "challengeAction"
+  | "creatureCombat"
+  | "pvpCombat"
+  | "bossReferenceList"
+  | "guideToFirstGame";
 
 const ReferenceList = () => {
-  return (
-    <>
-      <span>Reference</span>
-    </>
-  );
+  const [contents, setContents] = useState<Contents>("list");
+
+  const BackButton = () => <FloatingBackButton onClick={() => setContents("list")}>←</FloatingBackButton>;
+
+  switch (contents) {
+    case "referenceSheet":
+      return <FullWidthImageWithBackButton renderBackButton={BackButton} path={referenceSheetPath} />;
+    case "monsterReference":
+      return <FullWidthImageWithBackButton renderBackButton={BackButton} path={monsterReferencePath} />;
+    case "challengeAction":
+      return <FullWidthImageWithBackButton renderBackButton={BackButton} path={challengeActionPath} />;
+    case "creatureCombat":
+      return <FullWidthImageWithBackButton renderBackButton={BackButton} path={creatureCombatPath} />;
+    case "pvpCombat":
+      return <FullWidthImageWithBackButton renderBackButton={BackButton} path={pvpCombatPath} />;
+    case "bossReferenceList":
+      return <ListBossReference renderBackButton={BackButton} />;
+    case "guideToFirstGame":
+      return <FullWidthImageWithBackButton renderBackButton={BackButton} path={guideToFirstGamePath} />;
+    case "list":
+    default:
+      return (
+        <>
+          <FullWidthListContainer onClick={() => setContents("referenceSheet")}>Reference Sheet</FullWidthListContainer>
+          <FullWidthListContainer onClick={() => setContents("monsterReference")}>
+            Monster Reference Sheet
+          </FullWidthListContainer>
+          <FullWidthListContainer onClick={() => setContents("challengeAction")}>
+            Challenge Action Example
+          </FullWidthListContainer>
+          <FullWidthListContainer onClick={() => setContents("creatureCombat")}>
+            Creature Combat Example
+          </FullWidthListContainer>
+          <FullWidthListContainer onClick={() => setContents("pvpCombat")}>PvP Combat Example</FullWidthListContainer>
+          <FullWidthListContainer onClick={() => setContents("bossReferenceList")}>
+            Boss Reference Cards
+          </FullWidthListContainer>
+          <FullWidthListContainer onClick={() => setContents("guideToFirstGame")}>
+            Guide to First Game
+          </FullWidthListContainer>
+        </>
+      );
+  }
+};
+
+export const FloatingBackButton = styled.div`
+  position: absolute;
+  height: 64px;
+  width: 64px;
+  color: red;
+  font-size: 56px;
+  line-height: 56px;
+  font-weight: 900;
+  text-align: center;
+`;
```

Comment:
/nit: Can we make this consistent with the close button in the carousel? (X in the top right)

Reply:
yeah make this consistent in a follow up PR